Features/Stories/FieldTypes/TextlineFieldType.feature

```diff
@@ -0,0 +1,246 @@
+        When I create a content of this type
+        And I set "Test text" as the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript @common
+    Scenario: Creating a text line Field with an empty value works
+        Given a Content Type with an "text line" Field exists
+        When I create a content of this type
+        And I set an empty value as the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript @edge
+    Scenario: Creating a required text line Field fails validation when using an empty value
+        Given a Content Type with a required "text line" Field exists
+        When I create a content of this type
+        And I set an empty value as the Field Value
+        And I publish the content
+        Then Publishing fails with validation error message "This field is required"
+
+    @javascript @common
+    Scenario: Creating a valid text line Field works when using a value within limited character scope
+        Given a Content Type with an "text line" Field exists with Properties:
+            | Validator                | Value |
+            | minimum length validator | 2     |
+            | maximum length validator | 4     |
+        When I create a content of this Type
+        And I set "LOL" as the Field Value
+        And I publish the content
+        Then the Content is successfully published
+
+    @javascript @edge
+    Scenario: Creating an invalid text line Field fails validation when using a value smaller than minimum character limit allowed
+        Given a Content Type with an "text line" Field exists with Properties:
+            | Validator                | Value |
+            | minimum length validator | 2     |
+            | maximum length validator | 4     |
+        When I create a content of this Type
+        And I set "X" as the Field Value
+        And I publish the content
+        Then Publishing fails with validation error message "The value should have at least 2 characters"
+
+    @javascript @edge @notWorking
+    Scenario: Creating an invalid content type with a text line Field fails validation when using a maximum bigger than the minimum character limit
+        Given I am on the "Content types" page
+        And I click on the "Content" link
+        When I click at "Create a content type" button
+        And I fill form with:
+            | Field      | Value |
+            | Name       | Test  |
+            | Identifier | test  |
+        And I add a field type "Text line" with:
+            | Field           | Value |
+            | Name            | Text  |
+            | Identifier      | text  |
+            | Minimum length  | 4     |
+            | Maximum length  | 2     |
+        And I click at "OK" button
+        Then Publishing fails with validation error message "Form did not validate. Please review errors below."
```

Comment:
I'm a bit bothered by *Publishing*, a Content Type is not really published (even if we have a draft status for content types, but that's a more internal detail, AFAIK, this is not exposed to the user). Here I would write something like:
`Then the Content Type is not stored` (I'm unsure whether the error message should be tested)